packages/launcher/src/renderer/launcher/Launcher.js

```diff
@@ -397,6 +403,16 @@ export default class App extends Component<{}, State> {
       </View>
     )
   }
+
+  render() {
+    return (
+      <MFThemeProvider theme={THEME.morpheus}>
+        <SCThemeProvider theme={THEME.styled}>
```

Comment:
Do we need to have 2 theme providers? That seems quite wasteful, couldn't we leverage styled-components' theming solution without providing our own?

Reply:
I included styled components ThemeProvider in Morpheus UI